openbb_terminal/cryptocurrency/onchain/topledger_model.py

```diff
@@ -0,0 +1,102 @@
+"""Topledger model"""
+__docformat__ = "numpy"
+
+import logging
+from typing import Any, Optional, Tuple
+
+import pandas as pd
+
+from openbb_terminal.decorators import log_start_end
+from openbb_terminal.helper_funcs import request
+from openbb_terminal.rich_config import console
+
+logger = logging.getLogger(__name__)
+
+MAPPING = {
+    "tensor": {
+        "api_key": "8eSjJnSuOC7uLWkrTQoWADStkNS2ZIy94pH5CsNn",
+        "queries": [
+            {"id": 846, "slug": "daily-transactions"},
+            {"id": 847, "slug": "weekly-transactions"},
+            {"id": 848, "slug": "monthly-transactions"},
+            {"id": 849, "slug": "daily-users"},
+            {"id": 852, "slug": "monthly-users"},
+            {"id": 851, "slug": "weekly-users"},
+            {"id": 1092, "slug": "daily-gmv-sol"},
+            {"id": 1104, "slug": "weekly-gmv"},
+            {"id": 1105, "slug": "monthly-gmv"},
+            {"id": 1110, "slug": "daily-tvl"},
+            {"id": 1111, "slug": "weekly-tvl"},
+            {"id": 1112, "slug": "monthly-tvl"},
+            {"id": 1115, "slug": "top-traders-by-gmv"},
+        ],
+    }
+}
+
+
+@log_start_end(log=logger)
+def make_request(org_slug=None, query_slug=None) -> Tuple[Optional[int], Any]:
```

Comment:
All functions need docstrings that have:
"""Description of what the function actually does

Parameters
-----------
parameter: Type
    description of the parameter

Returns
--------
response: Type
    What type of object is returned.

Examples
----------
>>> a working example of the function.